app/views/chapters/show.html.erb

```diff
@@ -1,5 +1,5 @@
-<h2 class="text-center mt-3"><%= t ".story" %>: <%= @chapter.story.name %></h2>
+<h2 class="text-center mt-3"><%= @chapter.story.name %></h2>
 <h4 class="text-center"><%= @chapter.name %></h4>
 <div class="mt-4" id="chapter-content">
-  <%= @chapter.content %>
+  <%= @chapter.content.html_safe %>
```

Comment:
html_safe để làm gì thế

Reply:
html_safe được gọi trên 1 chuỗi, chèn nội dung vào html mà không thêm bất kỳ kí tự thoát nào, hàm này chỉ nên dùng khi chắc chắn chuỗi là an toàn và không có nội dung độc hại, mã độc => html_safe không sử dụng cho nội dung được người dùng nhập vào. 
em chuyển sang dùng method **sanitize** rồi ạ. :v